.github/workflows/pull-request-workflow.yml

```diff
@@ -1,4 +1,4 @@
-name: Pull Request Workflow
+name: Viperlight
 
 on: 
   pull_request:
```

Comment:
This currently only runs on pull_requests. Should it be updated to run on pushes as well?

Reply:
I don't see a reason not to

Reply:
k. updating